apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2/TranslationClientImpl.cs

```diff
@@ -132,12 +132,19 @@ public override IList<TranslationResult> TranslateHtml(IEnumerable<string> htmlI
         }
 
         /// <inheritdoc />
-        public override IList<Detection> DetectLanguage(string text)
+        public override Detection DetectLanguage(string text) =>
```

Comment:
The async implementation is essentially all boiler-plate, it's not adding any behaviour.
So I think it'd be better in the base abstract class.